CMakeLists.txt

```diff
@@ -14,19 +25,34 @@ include(DependencyCheck)
 include(Version)
 include(FGInstallDirs)
 
+if(USE_HUNTER)
+    hunter_add_package(glbinding)
+    hunter_add_package(glm)
+    hunter_add_package(freetype)
+    hunter_add_package(Boost)
+
+    find_package(freetype CONFIG REQUIRED)
+else()
+    find_package(freetype REQUIRED)
+endif()
+
 find_package(Doxygen QUIET)
 find_package(glbinding REQUIRED)
 find_package(glm REQUIRED)
-find_package(FreeType)
-find_package(FontConfig QUIET)
+
+if(UNIX)
+    find_package(FontConfig REQUIRED)
+endif()
 find_package(X11 QUIET)
 find_package(FreeImage QUIET)
-find_package(Boost)
+find_package(Boost REQUIRED)
 
 option(BUILD_DOCS      "Build Documentation"               ${DOXYGEN_FOUND})
 option(BUILD_EXAMPLES  "Build Examples"                    ON)
 option(WITH_FREEIMAGE  "Use FreeImage to add support for
                         saving framebuffer to disk"        ${FreeImage_FOUND})
+option(WITH_FONTCONFIG  "Use FreeImage to add support for
```

Comment:
Where is this variable `WITH_FONTCONFIG` being used ?

Reply:
my mistake, I started to add an option then found out it wasn't really optional. Missed it on the cleanup.